src/raw/config-avalanche.js

```diff
@@ -0,0 +1,201 @@
+    },
+    tokenFactory: {
+      address: "0x4B9bd38593A26325218A16805C9D6D3651D9E8d0",
+      description: "Contract is used to reliably mint the synthetic tokens used by the float protocol.",
+      codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/TokenFactory.sol",
+    },
+    floatToken: {
+      address: "0x9fB133B2C5218D7DDb97422aa27c0eD04122c944",
+      symbol: "aFLT",
+      iconUrl:
+        "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/icons/alpha-float-token.svg",
+    },
+    floatCapital_v0: {
+      address: "0x865C1a2388E0E9C2D9b347D8DcAc617E056D16d9",
+      description: "A percentage of float token to accrue here for project development.",
+      codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/FloatCapital_v0.sol",
+    },
+    treasury_v0: {
+      address: "0x4e813624E1E0906e23fa22E2d120b4c4e2F89E4e",
+      description: "Allows the burning of FLT tokens for a proportional share of the value held in the treasury.",
+      codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/Treasury_v0.sol",
+    },
+    gems: {
+      address: "0xB779a4A28012e12Ff904754bBE72F60423AF0A28",
+      description: "Contract giving user GEMS, inspired by https://github.com/andrecronje/rarity/blob/main/rarity.sol",
+      codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/GEMS.sol",
+    },
+    gemCollectorNft: {
+      address: "0xEB73D5CbB26536F714785B7864806542f41ab9E5",
+      description: "NFTs for users who have accrued enough GEMS, a new NFT for each level of GEM accrual.",
+      codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/GemCollectorNFT.sol",
+    },
+  },
+  paymentTokens: [{
+    address: "0xd586e7f844cea2f87f50152665bcbc2c279d8d70",
+    isUSDBased: true,
+    decimals: 18,
+    symbol: "DAI",
+    iconUrl: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/icons/tokens/dai.svg",
+  }],
+  hiddenMarkets: [],
+  markets: [
+    {
+      index: 1,
+      name: "Avalanche Market",
+      description:
+        "Avax is the main token of the Avalanche blockchain and acts as the primary 'fuel' that powers all activity on it. Avax is used to perform actions on the network, such as sending transactions and interacting with dapps.",
+      iconUrl: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/tokens/avax/avax.png",
+      leverage: 2,
+      paymentTokenSymbol: "DAI",
+      symbol: "2AVAX",
+      launchTimestamp: 1639490400,
+      createdTimestamp: 1638805641,
+      yieldManager: {
+        address: "0x621cda10820555adAe8506eeC296cB9621E491Ff",
+        description: "Aave-based yield manager",
+        codeUrl: "https://github.com/Float-Capital/contracts/blob/master/contracts/YieldManagers/YieldManagerAaveBasic.sol",
```

Comment:
Nice touch :)

Reply:
Was thinking about adding links to ABIs as well, but will make that a task for the future